airbyte-webapp/src/hooks/services/Analytics/pageTrackingCodes.tsx

```diff
@@ -0,0 +1,33 @@
+export const PAGE_TRACKING_CODES = {
```

Comment:
We tend to use `enum`s for those places we plan to have "complete" lists of keys that should be used. Also we'd be able to change then the type of `useTrackPage` parameter type to only accept that enum.

Reply:
Also could we maybe re-export this type from the `src/hooks/services/Analytics/index.ts` file, so we can import it from there. We try to re-export types and methods that are supposed to be used outside that specific folder, and if it's not re-exported try to treat it as private to this folder.